From af0d0a466ef4199f74100cb2e2980af8d2b44ec8 Mon Sep 17 00:00:00 2001
From: Mark Verlinde <mark@havak.nl>
Date: Sun, 31 May 2020 07:06:54 +0200
Subject: [PATCH] efi use Mageia specific path name

Signed-off-by: Mark Verlinde <mark@havak.nl>
---
 include/config_distro_bootcmd.h | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/include/config_distro_bootcmd.h b/include/config_distro_bootcmd.h
index 0768afc4b5..cccdb8bc6d 100644
--- a/include/config_distro_bootcmd.h
+++ b/include/config_distro_bootcmd.h
@@ -92,9 +92,9 @@
 
 #ifdef CONFIG_EFI_LOADER
 #if defined(CONFIG_ARM64)
-#define BOOTEFI_NAME "bootaa64.efi"
+#define BOOTEFI_NAME "grubaa64.efi"
 #elif defined(CONFIG_ARM)
-#define BOOTEFI_NAME "bootarm.efi"
+#define BOOTEFI_NAME "grubarm.efi"
 #elif defined(CONFIG_X86_RUN_32BIT)
 #define BOOTEFI_NAME "bootia32.efi"
 #elif defined(CONFIG_X86_RUN_64BIT)
@@ -130,7 +130,7 @@
 			"bootefi bootmgr ${fdtcontroladdr};"              \
 		"fi;"                                                     \
 		"load ${devtype} ${devnum}:${distro_bootpart} "           \
-			"${kernel_addr_r} efi/boot/"BOOTEFI_NAME"; "      \
+			"${kernel_addr_r} efi/mageia/"BOOTEFI_NAME"; "      \
 		"if fdt addr ${fdt_addr_r}; then "                        \
 			"bootefi ${kernel_addr_r} ${fdt_addr_r};"         \
 		"else "                                                   \
@@ -165,9 +165,9 @@
 		"run boot_efi_binary\0"                                   \
 	"scan_dev_for_efi="                                               \
 		"if test -e ${devtype} ${devnum}:${distro_bootpart} "     \
-					"efi/boot/"BOOTEFI_NAME"; then "  \
+					"efi/mageia/"BOOTEFI_NAME"; then "  \
 				"echo Found EFI removable media binary "  \
-					"efi/boot/"BOOTEFI_NAME"; "       \
+					"efi/mageia/"BOOTEFI_NAME"; "       \
 				"run scan_dev_for_dtb; "                  \
 				"echo EFI LOAD FAILED: continuing...; "   \
 		"fi; "                                                    \
-- 
2.26.2

